products/vmware-cloud-director.md

```diff
@@ -0,0 +1,42 @@
+---
+title: VMware Cloud Director
+category: server-app
+tags: vmware
+iconSlug: vmware
+permalink: /vmware-vcd
+alternate_urls:
+-   /vmwarevcd
+releasePolicyLink: https://support.broadcom.com/group/ecx/productlifecycle
+eolColumn: General Support
+releaseDateColumn: true
+
+releases:
+-   releaseCycle: "10.6"
+    releaseDate: 2024-06-27
+    eol: 2026-06-27
+    latest: "10.6"
+    latestReleaseDate: 2024-06-27
+    link: https://docs.vmware.com/en/VMware-Cloud-Director/10.6/rn/vmware-cloud-director-106-release-notes/index.html
+-   releaseCycle: "10.5"
+    releaseDate: 2023-07-18
+    eol: 2025-07-18
+    latest: "10.5.1.1"
+    latestReleaseDate: 2024-03-07
+    link: https://docs.vmware.com/en/VMware-Cloud-Director/10.5.1.1/rn/vmware-cloud-director-10511-release-notes/index.html
+-   releaseCycle: "10.4"
+    releaseDate: 2022-07-14
+    eol: 2024-07-14
+    latest: "10.4.3"
+    latestReleaseDate: 2024-01-09
+    link: https://docs.vmware.com/en/VMware-Cloud-Director/10.4.3/rn/vmware-cloud-director-1043-release-notes/index.html
+---
+
+> [VMware Cloud Director](https://www.vmware.com/products/cloud-director.html) Operate and
+> manage your cloud service business with VMware Cloud Director,
+> the leading cloud service delivery platform.
+
+## [General Support](https://lifecycle.vmware.com/)
```

Comment:
https://lifecycle.vmware.com/ is not public anymore, the description must be updated to reflect that.